ee/agent/storage/sqlite/logstore_sqlite.go

```diff
@@ -0,0 +1,103 @@
+package agentsqlite
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+func (s *sqliteStore) getColumns() *sqliteColumns {
```

Comment:
This function feels like maybe it belongs in `keyvalue_store_sqlite.go` instead, since it applies to both tables? What do you think?

Reply:
yeah that probably makes more sense, I was torn about adding a new shared file but think we can probably wait on that. i'll move there for now!